cookbook/docker/example.md

```diff
@@ -0,0 +1,42 @@
+# Dockerize a Feathers application
+
+A Feathers application can be dockerized like any other Node.js app.
+
+## Create an app
+```sh
+$ mkdir feathers-app
+$ cd feathers-app/
+$ feathers generate app
+```
+
+## Add a Dockerfile
+```sh
+$ touch Dockerfile
+```
+
+### Dockerfile
+```
+FROM node:11-alpine
```

Comment:
Prefer an LTS or current version.

Reply:
are you ok with `node:alpine` or `node`?

Reply:
`node:lts-alpine` ?